pkg/repository/local_git.go

```diff
@@ -47,6 +47,7 @@ type LocalExec struct{}
 
 func (le *LocalExec) RunCommand(ctx context.Context, dir string, name string, arg ...string) ([]byte, error) {
 	c := le.createCommand(ctx, dir, name, arg...)
+	oktetoLog.Debugf("executing command: %s", strings.Join(c.Args, " "))
```

Comment:
As mention in the description, these are some debug logs that might help us to troubleshoot issue